app/models/insured/forms/self_term_or_cancel_form.rb

```diff
@@ -56,8 +56,13 @@ def self.for_aptc_update_post(attrs)
         return 'Action cannot be performed because of the overlapping plan years.' unless check_to_enable_tax_credit_btn(attrs)
 
         service = self_term_or_cancel_service(attrs)
-        service.update_aptc
-        'Tax credit updated successfully.'
+        status, error = service.update_aptc
+
+        if status
+          'Tax credit updated successfully.'
```

Comment:
is this displayed to the end user?

Reply:
it will display in UI, but for now this ticket does not require this change. I reverted this logic to previous one.